polytope_server/common/authentication/mongoapikey_authentication.py

```diff
@@ -39,18 +39,18 @@ class ApiKeyMongoAuthentication(authentication.Authentication):
     """
 
     def __init__(self, name, realm, config):
-
         self.config = config
         host = config.get("host", "localhost")
         port = config.get("port", "27017")
         collection = config.get("collection", "keys")
         username = config.get("username")
         password = config.get("password")
-        tls = config.get("tls", False) == True
+        srv = True if config.get("srv", False) else False
```

Comment:
```suggestion
        srv = bool(config.get("srv", False))
```
this should be equivalent but note that a non empty string e.g. "false" will still evaluate to True

Reply:
[1ba124b](https://github.com/MeteoSwiss-APN/polytope-server/pull/12/commits/1ba124b2b948ee4b365a0ea6cf029ee269ce2cca)